magma/lib/magma/query/predicate/model.rb

```diff
@@ -129,8 +129,15 @@ def select
       [ column_name.as(identity) ]
     end
 
-    def column_name
-      Sequel[alias_name][@model.identity.column_name]
+    def column_name(attribute = @model.identity)
```

Comment:
These improved methods make selecting by an attribute on the model easier, which is necessary for changes made to `question.rb`